test/Fixtures/CustomerFixtureTrait.php

```diff
@@ -122,12 +124,20 @@ public function getMinimalRegisteredB2bCustomer(): Customer
      */
     public function getMaximalRegisteredB2bCustomer(): Customer
     {
-        return $this->getMinimalRegisteredB2bCustomer()
+        $owner = (new CompanyOwner())
+            ->setFirstname('firstname')
+            ->setLastname('lastname')
+            ->setBirthdate('1999-01-01');
+
+        $customer = $this->getMinimalRegisteredB2bCustomer()
             ->setShippingAddress($this->getShippingAddress())
             ->setSalutation(Salutations::MR)
             ->setMobile('+49172123456')
             ->setPhone('+4962216471100')
             ->setBillingAddress($this->getBillingAddress());
+
+        $customer->getCompanyInfo()->setOwner($owner);
```

Comment:
Old tests should not be changed.
This parameter is present for upl invoice only.

Reply:
This is about updating the maximum customer fixture which i think should be updated. This is independent of a specific payment method.
This fixture is used for customer tests only so far.

Reply:
but you do not make sure "old" implementations still work.